src/proxy/processors/push-action/writePack.js

```diff
@@ -7,19 +7,21 @@ const exec = async (req, action) => {
     const cmd = `git receive-pack ${action.repoName}`;
     step.log(`executing ${cmd}`);
 
-    const content = execSync(`git receive-pack ${action.repoName}`, {
+    const content = spawnSync('git', ['receive-pack', action.repoName], {
```

Comment:
See my previous comment on `spawnSync` use and whether we're OK blocking here or not. Probably needs the same `promisify(execFile)` pattern.

Reply:
Same as my comments above, thoughts?